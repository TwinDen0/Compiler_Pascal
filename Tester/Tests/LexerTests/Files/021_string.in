'aa '#100' aa'