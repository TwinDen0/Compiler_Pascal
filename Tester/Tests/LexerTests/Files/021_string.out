1,	1,	STRING,	d,	#100
EndFile